test/behavior/maximum_minimum.zig

```diff
@@ -336,3 +336,18 @@ test "@min/@max of signed and unsigned runtime integers" {
     try expectEqual(x, @min(x, y));
     try expectEqual(y, @max(x, y));
 }
+
+test "@min resulting in u0" {
+    if (builtin.zig_backend == .stage2_aarch64) return error.SkipZigTest; // TODO
+    if (builtin.zig_backend == .stage2_arm) return error.SkipZigTest; // TODO
+    if (builtin.zig_backend == .stage2_sparc64) return error.SkipZigTest; // TODO
+    if (builtin.zig_backend == .stage2_riscv64) return error.SkipZigTest;
```

Comment:
Fwiw, this test will pass on `stage2_riscv64`.

Reply:
Ah, okay -- sorry, I was just pasting skips from a similar test.